### ISSUE TITLE
'SfmRecordMapperProvider' How to add alias by pojo type

### ISSUE THREAD

**Issue by cn-src** [arnaudroger/SimpleFlatMapper#629]

**Comment by arnaudroger** [comment]
interesting, I don't think it's possible at the minute.
the way to deal with that would be to use the @Column JPA annotation javax.persistence.Column on the class
there should definitelty be another way to do that will see how we can add that.

**Comment by cn-src** [comment]
@arnaudroger 
Does SimpleFlatMapper plan to provide built-in `@Column` or `@SfmAlias` annotations?

**Comment by arnaudroger** [comment]
that would be very easy to add will get that with overall fix.

for the @column of jpa all you need is add that dependency 

		<dependency>
			<groupId>org.eclipse.persistence</groupId>
			<artifactId>javax.persistence</artifactId>
			<version>2.2.0</version>
</dependency>

**Comment by arnaudroger** [comment]
released in 6.7.0 new @Column annotation and alias per type 
should be available in maven central in 30 minutes

```java
SfmRecordMapperProvider recordMapperProvider = 
                SfmRecordMapperProviderFactory
                        .newInstance()
                        .addAliasForType(Bar.class, "barId", "id")
                        .addAliasForType(Zoom.class, "zoomId", "id")
                        .newProvider();
```
